Projects/App/Sources/Screens/ProfileScreen/ProfileState.swift

```diff
@@ -112,9 +111,8 @@ public final class ProfileStore: StateStore<ProfileState, ProfileAction> {
     public override func reduce(_ state: ProfileState, _ action: ProfileAction) -> ProfileState {
         var newState = state
         switch action {
-        case .setProfileState(let name, let charity, let monthlyNet):
+        case .setProfileState(let name, let monthlyNet):
             newState.memberFullName = name
-            newState.memberCharityName = charity
```

Comment:
can we also remove memberCharityName from state since we are not settings it ( old users will still have it set and will not change overtime) ?

Reply:
Yes I think we could do that. Good catch 👍